compiler/modules/front/token.bal

```diff
@@ -250,77 +282,81 @@ class Tokenizer {
                 return [DECIMAL_NUMBER, digits];
             }
             else if ch == "\"" {
-                string content = "";
-                while true {
+                return self.tokenizeStr();
```

Comment:
Had to reduce the size of this method due to a known bug in jBallerina  ballerina-platform/ballerina-lang#29137.
Github diff is confusing, you may append `&w=1` to the URL to ignore white space changes in UI.

Reply:
Diffs are pretty unintelligible but the function needs splitting up anyway.